packages/prism-theme-odyssey/src/utils/CustomPropertyInspector.js

```diff
@@ -0,0 +1,122 @@
+const Storage = window.localStorage;
+
+class CustomPropertyInspector {
+  constructor({propertiesElement, inspectorElement}) {
+    this.properties = this.setCustomProperties(propertiesElement);
+
+    this.renderInspector()
+  }
+
+  setCustomProperties(element) {
+    let prop
+    const styles = getComputedStyle(element)
+    const properties = {}
+
+    for (prop in styles) {
+      const name = styles[prop];
+      const value = Storage.getItem(name) || styles.getPropertyValue(name);        
+
+      if (styles.hasOwnProperty(prop) && styles[prop].includes('--')) { 
```

Comment:
```suggestion
      if (styles.hasOwnProperty(prop) && styles[prop].startsWith('--')) { 
```

This works! But consider a more explicit check for CSS custom properties as they should always be prefix with double hyphens. All that said, I don't believe double hyphens would appear elsewhere in other style names but its nice to be explicit when we can and prevent unexpected behavior